drivers/espi/espi_it8xxx2.c

```diff
@@ -1870,6 +1876,9 @@ static int espi_it8xxx2_init(const struct device *dev)
 			DEVICE_DT_INST_GET(0), 0);
 	irq_enable(IT8XXX2_ESPI_VW_IRQ);
 
+	/* Reset PLTRST# virtual wire signal during eSPI reset */
+	vw_reg->VWCTRL2 |= IT8XXX2_ESPI_VW_RESET_PLTRST;
```

Comment:
In the [ChromeOS code](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/HEAD:src/platform/ec/chip/it83xx/clock.c;l=299), it looks like this bit is only set for boards using the [IT8320](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/HEAD:src/platform/ec/chip/it83xx/config_chip_it8320.h;l=72).  Is this change needed on the IT81202/IT81302?

I also don't see this bit defined in the IT81302 datahseet.

Reply:
> In the [ChromeOS code](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/HEAD:src/platform/ec/chip/it83xx/clock.c;l=299), it looks like this bit is only set for boards using the [IT8320](https://source.chromium.org/chromiumos/chromiumos/codesearch/+/HEAD:src/platform/ec/chip/it83xx/config_chip_it8320.h;l=72). Is this change needed on the IT81202/IT81302?
> 
Yes. IT81302 does not require the eSPI reset workaround.
But IT81302 also need to enable this bit so that PLTRST# can be reset by eSPI reset.
We should correct ChromeOS code:
To make VWCTRL2 configuration is out of the IT83XX_ESPI_RESET_MODULE_BY_FW option.

> I also don't see this bit defined in the IT81302 datahseet.

The description will be added in the next datasheet release.